### ISSUE TITLE
Subscribing to Topic

### ISSUE THREAD

**Issue by thedumbtechguy** [Atrix1987/gcm-android-client#3]
I've implemented your library (which is awesome btw) and wonder how to subscribe to topics.

I've seen that you have to send an intent, but can you demonstrate what I need to send exactly? Sorry if this sounds noobish.

**Comment by Atrix1987** [comment]
For subscribing to a topic use the following code:

`GcmHelper.getInstance().subscribeTopic(context, topicName);`

**Don't forget to update to version 1.0.2.** 

Also make sure you are calling the init method that i.e. something like the following

```
GcmHelper.getInstance()
        .setAuthorizedEntity("889908101771")
        .addRegistrationCallback(getApplicationContext(), new CustomRegistrationListener(), true)
        .addOnMessageReceivedCallback(new CustomMessageReceivedListener())
        .init(getApplication());
```

**Comment by thedumbtechguy** [comment]
So simple.
I can't believe I missed it.

Thanks.

On Fri, May 20, 2016 at 1:37 PM, Abhishek Nandi notifications@github.com
wrote:

> Closed #3 https://github.com/Atrix1987/gcm-android-client/issues/3.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/Atrix1987/gcm-android-client/issues/3#event-667147468

## 

Regards,
Stefan Froelich.
